app/commands/actions.html

```diff
@@ -284,8 +284,8 @@ <h4 id="dblclick"><a href="https://on.cypress.io/dblclick">.dblclick()</a></h4>
           <h4 id="rightclick"><a href="https://on.cypress.io/rightclick">.rightclick()</a></h4>
           <p>To right click a DOM element, use the <a href="https://on.cypress.io/rightclick"><code>.rightclick()</code></a>
             command.</p>
-          <pre><code class="javascript">cy.get('.action-div').rightclick().should('not.be.visible')
-                cy.get('.action-input-hidden').should('be.visible')</code></pre>
+          <pre><code class="javascript">cy.get('.rightclick-action-div').rightclick().should('not.be.visible')
+                cy.get('.rightclick-action-input-hidden').should('be.visible')</code></pre>
```

Comment:
I also noticed the spacing is not correct on the deployed page.

```suggestion
cy.get('.rightclick-action-input-hidden').should('be.visible')</code></pre>
```